temporian/core/event_set_ops.py

```diff
@@ -1093,6 +1093,55 @@ def enumerate(self: EventSetOrNode) -> EventSetOrNode:
 
         return enumerate(self)
 
+    def fft(
+        self: EventSetOrNode,
+        num_events: int,
+        window: Optional[str] = None,
+        num_spectral_lines: Optional[int] = None,
+    ) -> EventSetOrNode:
```

Comment:
The most common case is to have a `hop_size` parameter as well, usually a half or quarter of `window_size`.
Maybe it should be added in a future PR but the `num_events` name sounds weird to me. Maybe `window_events` or `window_n`? The fact that it's an `int` already makes pretty clear that this is not duration in seconds.

Reply:
- Adding "hop_size" argument.
- For the window_size, I want to make it super clear that this is not a window_length (both for code writer and readers). While I don't think "num_events" is very good, I think "window_size" is too close to "window_length".

WDYT?